src/components/tabController/TabBar.js

```diff
@@ -54,6 +55,10 @@ class TabBar extends PureComponent {
      * Show Tab Bar bottom shadow
      */
     enableShadow: PropTypes.bool,
+    /**
+     * custom shadow style
+     */
+    shadowStyle: RNText.propTypes.style,
```

Comment:
Why are you taking shadow style from RNText?

Reply:
I just saw it on `labelStyle `and `selectedLabelStyle` so I thought it is the relevant type of style prop.. 
Should it be `ViewPropTypes.style`?

Reply:
Yes 👍